nsxt/resource_nsxt_policy_dhcp_relay.go

```diff
@@ -22,7 +22,7 @@ func resourceNsxtPolicyDhcpRelayConfig() *schema.Resource {
 		Update: resourceNsxtPolicyDhcpRelayConfigUpdate,
 		Delete: resourceNsxtPolicyDhcpRelayConfigDelete,
 		Importer: &schema.ResourceImporter{
-			State: nsxtPolicyPathResourceImporter,
+			State: getFriendlyPolicyPathOrIDResourceImporter(getMultitenancyPathExample("/infra/dhcp-relays/[dhcp-relay]")),
```

Comment:
I recall we discussed you would look into extracting the example path from the API spec. Was that for generation?

Reply:
Correct, generator will fill this up